src/MessagePack.UnityClient/Assets/NuGet.config

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
Since this is generated by NuGetForUnity, it doesn't have much deep meaning.
This is also for development purposes, so it's not included when users reference it.